src/Android/Xamarin.Android/Resources/layout/ApiKeyPrompt.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
```

Comment:
I recommend adding some small amount of padding so that the text isn't running to the edge of the screen